__tests__/html/activityStatusTelemetry.ignoreStatusChange.incomingActivity.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <link href="/assets/index.css" rel="stylesheet" type="text/css" />
+    <script crossorigin="anonymous" src="/test-harness.js"></script>
+    <script crossorigin="anonymous" src="/test-page-object.js"></script>
+    <script crossorigin="anonymous" src="/__dist__/webchat-es5.js"></script>
+  </head>
+  <body>
+    <div id="webchat"></div>
+    <script>
+      run(
+        async function () {
+          const store = testHelpers.createStore();
+
+          const directLine = testHelpers.createDirectLineEmulator(store);
+          const telemetryEventTracking = [];
+          WebChat.renderWebChat(
+            {
+              directLine,
+              store,
+              styleOptions: {
+                groupTimestamp: 60000,
+                sendTimeout: 5000
+              },
+              onTelemetry: (event) => {
+                if (event?.name === 'send-status:change') {
+                  telemetryEventTracking.push(event);
+                }
+              },
+            },
+            document.getElementById('webchat')
+          );
+
+          await pageConditions.uiConnected();
+
+          await directLine.emulateIncomingActivity('Aloha!');
+
+          await pageConditions.became(
+            'send message successfully',
+            () => pageElements.activityStatuses()[0]?.innerText.indexOf('Just now') !== -1,
+            1000
+          );
+
+          expect(telemetryEventTracking.length).toEqual(0);
```

Comment:
```suggestion
          // THEN: No telemetry event should be emitted.
          //            This is because `send-status:change` only emit for outgoing activities, not incoming activities.
          expect(telemetryEventTracking.length).toEqual(0);
```

Reply:
updated